internal/helpers.go

```diff
@@ -147,6 +147,15 @@ func GetSrcFkFromId(fks []schema.ForeignKey, fkId string) (schema.ForeignKey, er
 	return schema.ForeignKey{}, fmt.Errorf("foreign key not found")
 }
 
+func GetCheckConstraintFromId(checkConstraints []ddl.CheckConstraint, checkConstraintName string) (ddl.CheckConstraint, error) {
```

Comment:
since this method is used only in test assertions , it might be better to move it to `test_util.go`